test/controllers/api/registry/registry_proxies_controller_test.rb

```diff
@@ -267,6 +267,23 @@ def setup_permissions
         assert_equal "dev_label-published_dev_view-puppet_product-busybox", body["results"][0]["name"]
       end
 
+      it "show unauthenticated repositories for head requests" do
```

Comment:
I kind of feel this test is trying to do too much. Would it make sense to break it down to two tests? One would check that HEAD request works and another would check that Manifest Version 2 Schema 2 correctly rewrites Manifest's `mediaType`.

Reply:
updated. I think the media type is tested else where in other tests in the same file, so decided to leave that out.

Reply:
Correct, there are other tests here that exercise `mediaType`. Thanks.